cluster/ci/monitoring/mixins/config.libsonnet

```diff
@@ -37,6 +37,11 @@
         channel: '#build-cop-alerts',
         notify: 'bparees',
       },
+      'kni-cnf': {
+        team: 'kni-cnf',
+        channel: '#forum-kni-cnf',
+        notify: '',
```

Comment:
NIP: do you have slack team-alias for notification?

Reply:
no, not right now :/ I'll have to update this later I suppose.